source/main/windows/main.js

```diff
@@ -37,7 +40,7 @@ export const createMainWindow = () => {
 
   window.loadURL(`file://${__dirname}/../renderer/index.html`);
   window.on('page-title-updated', event => { event.preventDefault(); });
-  window.setTitle(`Daedalus (${environment.DAEDALUS_VERSION || 'dev'})`);
+  window.setTitle(`Daedalus (${environment.build})`);
```

Comment:
cc @nikolaglumac

Reply:
webpack didnt detect this, because its indirectly accessing the env vars
but the environment object already has a dedicated field for this value, and default value logic
that resulted it in requiring a `$DAEDALUS_VERSION` at runtime to have any effect